examples/deadline/All-In-AWS-Infrastructure-SEP/ts/bin/app.ts

```diff
@@ -12,14 +12,6 @@ import * as cdk from 'aws-cdk-lib';
 import { SEPStack } from '../lib/sep-stack';
 import { config } from './config';
 
-// ------------------------------ //
-// --- Validate Config Values --- //
-// ------------------------------ //
-
-if (config.deadlineClientLinuxAmiMap === {['region']: 'ami-id'}) {
```

Comment:
This currently isn't useful because:
1. JavaScript comparisons of objects are done by reference, not by value
2. Our default value doesn't match this anyway
3. Stronger/full validation will still be done by CDK or CloudFormation